Nasal/RADAR/rwr-only.nas

```diff
@@ -136,7 +136,7 @@ props.globals.getNode("instrumentation/rwr/serviceable",1).setBoolValue(1);
 
 # start generic radar system
 var baser = AIToNasal.new();
-var omni = OmniRadar.new(1.0, 150, 55);
+var omni = OmniRadar.new(1.0, 150, -1);
```

Comment:
why is the TP range being changed to be -1nm?

Reply:
Means that RWR doesn't detect M3 transponders, like is now the case on the majority of the OPRF fleet. It wouldn't be  able to determine an aircraft type from a transponder signal if it could do that irl, and I don't think any RWR could pick up tp anyway.